SimTracker/TrackerHitAssociation/interface/ClusterTPAssociation.h

```diff
@@ -0,0 +1,68 @@
+#ifndef SimTracker_TrackerHitAssociation_ClusterTPAssociation_h
+#define SimTracker_TrackerHitAssociation_ClusterTPAssociation_h
```

Comment:
I'm not sure if this is the best package for this event product, but the dictionary for the preceding `ClusterTPAssociationList` was defined here.